dev/tasks/conda-recipes/arrow-cpp/bld.bat

```diff
@@ -20,6 +20,7 @@ cmake -G "Ninja" ^
       -DCMAKE_BUILD_TYPE=release ^
       -DARROW_PYTHON:BOOL=ON ^
       -DARROW_MIMALLOC:BOOL=ON ^
+      -DARROW_DATASET:BOOL=ON ^
```

Comment:
Should also add ARROW_CSV, ARROW_COMPUTE, ARROW_FILESYSTEM and ARROW_JSON explicitly, no?

Reply:
These non-optional (for Python) components are enabled automatically when ARROW_PYTHON=ON

https://github.com/apache/arrow/blob/master/cpp/CMakeLists.txt#L286